virttest/utils_libvirt/libvirt_disk.py

```diff
@@ -160,6 +160,16 @@ def create_disk(
         )
 
 
+def _all_disk_targets():
+    devs = []
+    vmx = vm_xml.VMXML.new_from_dumpxml(vm_name)
+    for disk in (vmx.get_devices("disk") or []):
+        tgt = (disk.target or {}).get("dev")
+        if tgt:
+            devs.append(tgt)
+    return devs
```

Comment:
Before return, please add like below for debugging
`LOG.debug("Target devs are %s", devs)`